.github/workflows/run-sugar-cli-test.yml

```diff
@@ -0,0 +1,69 @@
+name: Run Sugar CLI Test 2
+
+on:
+  workflow_dispatch:
+
+env:
+  SOLANA_VERSION: 1.9.22
+  RUST_TOOLCHAIN: stable
+
+jobs:
+  run-sugar-cli-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/install-linux-build-deps
+      - uses: ./.github/actions/install-solana
+        with:
+          solana_version: ${{ env.SOLANA_VERSION }}
+      - uses: ./.github/actions/install-rust
+        with:
+          toolchain: ${{ env.RUST_TOOLCHAIN }}
+
+      # todo: install from https://sugar.metaplex.com/install.sh after a new release that includes this commit
+      # https://github.com/metaplex-foundation/sugar/commit/875fe740fca6ac8d7dddd4d4c5bfaee363262635
+      - name: Pull in test script from Sugar repository (outside of MPL)
+        shell: bash
+        run: |
+          cd .. && git clone https://github.com/metaplex-foundation/sugar.git && cd sugar
+          cargo install --path ./ --force
+
+      - run: solana-keygen new -o $GITHUB_WORKSPACE/kp --no-bip39-passphrase
+        shell: bash
+
+      - run: solana config set --url http://127.0.0.1:8899 --keypair $GITHUB_WORKSPACE/kp
+        shell: bash
+
+      # `sugar-cli-test.amman.js` specifies how to load candy machine and token metadata bytes into the local validator
+      - name: Install and start amman with custom config
+        shell: bash
+        run: |
+          cd candy-machine/js
+          YARN_ENABLE_IMMUTABLE_INSTALLS=false yarn && npm install --location=global @metaplex-foundation/amman
+          cat $GITHUB_WORKSPACE/.github/.config/sugar-cli-test.amman.js > .ammanrc.js
```

Comment:
Why isn't this `.ammanrc.js` already in the right place? Isn't it needed to run tests locally?

Reply:
Good point - I thought that we needed to be within an existing directory with an `.ammanrc.js`, but it looks like the relative import for the base `ammanrc.js` works + we install amman globally. I will remove the cd + cat.